src/tchap/components/views/common/Gaufre.tsx

```diff
@@ -72,7 +72,7 @@ const TchapGaufre: React.FC<{
     return (
         <>
             <AccessibleButton
-                className={classNames(["mx_QuickSettingsButton", { expanded: !isPanelCollapsed }, "lasuite-gaufre-tchap", "lasuite-gaufre-mask"])}
+                className={classNames(["mx_QuickSettingsButton", { expanded: !isPanelCollapsed }, "lasuite--gaufre-borderless", "lasuite-gaufre-tchap", "lasuite-gaufre-mask"])}
```

Comment:
the "borderless" classname should be appended on something that is a parent of the popup content, that's why it's mentioned to rather add it to html or body in the docs ;) here it doesn't do anything.